### PR TITLE
new package: org-roam

### PR DESCRIPTION
**Pull request by ArsenArsen**
and emacsql.

note that this code has only been given a very light review, as I'm writing late and in haste.  emacsql also has tests that I'm not sure we can integrate too well.

this package appears to work at a glance, but I'm unsure if I missed anything - still getting comfortable writing packages for elisp code

TIA, have a lovely night

### CODE REVIEW

**Comment by ArsenArsen** [comment]
that assignee seems wrong.. the correct one should be @gentoo/emacs

**Comment by ulm** [comment]
> that assignee seems wrong.. the correct one should be @gentoo/emacs

@gentoo/gnu-emacs really. :smile:

File: app-emacs/org-roam/org-roam-2.2.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	app-emacs/dash
+	app-emacs/magit
+	app-emacs/emacsql[sqlite]
+"
+DEPEND="${RDEPEND}"
```
Comment (ulm) [inline app-emacs/org-roam/org-roam-2.2.2.ebuild:24]: Should be `BDEPEND` here, because byte-compilation takes place using Emacs on the build host.

**Comment by ArsenArsen** [comment]
> > that assignee seems wrong.. the correct one should be @gentoo/emacs
> 
> @gentoo/gnu-emacs really. :smile:
> 

ah, my bad.  I only saw {x,}emacs, and I saw the people I'd expect in the emacs team, so I didn't realize there was a gnu-emacs

File: app-emacs/org-roam/org-roam-2.2.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+NEED_EMACS=26.1
+inherit elisp
+
+DESCRIPTION="Rudimentary Roam replica with Org-mode"
+HOMEPAGE="https://github.com/org-roam/org-roam"
+SRC_URI="
```
Comment (xgqt) [inline app-emacs/org-roam/org-roam-2.2.2.ebuild:11]: org-roam -> ${PN}

maybe add live handling? :)

**Pull request closed without merging**